### DISCUSSION TITLE
yank code block

### DISCUSSION THREAD

**Discussion by ryoppippi** [olimorris/codecompanion.nvim#327] [Ideas]
If there is a keymap to yank code block just like CopilotChat.nvim, it makes my life easier

**Comment by olimorris** [comment]
Great suggestion. I'll add this to the roadmap

**Comment by olimorris** [comment]
@ryoppippi I've added this as of `v8.7.0`. If you do `gy` in the chat buffer it will yank the last codeblock. In time I'd like to add a feature where you yank the last codeblock or the codeblock the cursor is in.

**Reply by ryoppippi** [reply]
Thank you for your quick work! Brilliant. It works!!
